### ISSUE TITLE
Unexpected token '=>'

### ISSUE THREAD

**Issue by jimmynames** [barbajs/barba#491]
Attempting to add Barba to a simple html, jquery project and upon adding this code

`
barba.init({
    transitions: [{
        leave({ current, next, trigger }) {
            // do something with `current.container` for your leave transition
            // then return a promise or use `this.async()`
        },
        enter({ current, next, trigger }) {
            // do something with `next.container` for your enter transition
            // then return a promise or use `this.async()`
        }
    }]
});

// dummy example to illustrate rules and hooks
barba.init({
    transitions: [{
        name: 'dummy-transition',

        // apply only when leaving `[data-barba-namespace="home"]`
        from: 'home',

        // apply only when transitioning to `[data-barba-namespace="products | contact"]`
        to: {
            namespace: [
                'products',
                'contact'
            ]
        },

        // apply only if clicked link contains `.cta`
        custom: ({ current, next, trigger })
            => trigger.classList && trigger.classList.contains('cta'),

        // do leave and enter concurrently
        sync: true,

        // available hooks…
        beforeOnce() { },
        once() { },
        afterOnce() { },
        beforeLeave() { },
        leave() { },
        afterLeave() { },
        beforeEnter() { },
        enter() { },
        afterEnter() { }
    }]
});`

I'm getting this error 

There doesn't seen to be any example of this new version being used in the native client rather than applications with npm etc..

any help apprecated

**Comment by xavierfoucrier** [comment]
@jimmynames,

Please update your issue to explain it with more consistency and better code syntax.
Thanks.

**Comment by elebumm** [comment]
```js
custom({ current, next, trigger }) {
        trigger.classList && trigger.classList.contains('cta')
}
```

It seems like the fat arrow function in your `custom` isn't working. How about when it's done like this instead?

**Comment by olets** [comment]
I ran into this too. PR #495 has the fix

**Comment by xavierfoucrier** [comment]
Thanks for the issue. This will be fixed in the next documentation review #509.
